nixos/modules/services/x11/imwheel.nix

```diff
@@ -0,0 +1,63 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.xserver.imwheel;
+in
+  {
+    options = {
+      services.xserver.imwheel = {
+        enable = mkEnableOption "IMWheel service";
+
+        options = mkOption {
+          type = types.listOf types.str;
+          default = ["-d" "-b 45" "--kill"];
```

Comment:
nit: why mix long and short argument forms?

Reply:
good poit, just a negligence on my part. I would use long forms as those are more descriptive, agree?

Reply:
I like long form also.